### PR TITLE
feat: Increase size of logo

### PR DESCRIPTION
**Pull request by Klaas-Ritense**
After request from rutger

Screenshot 2023-01-09 at 09 11 43

Screenshot 2023-01-06 at 16 55 26

### CODE REVIEW

**Comment by rutger-h** [comment]
It doesn't have the 8px padding as we tested, both not on test and in your preview.  The logo doesn't have 8px whitespace to pand bottom as far as I can see?

File: projects/valtimo/components/src/lib/components/topbar/topbar.component.scss [hunk 1]
```diff
@@ -36,9 +36,9 @@
```
Comment (Klaas-Ritense) [inline projects/valtimo/components/src/lib/components/topbar/topbar.component.scss:36]: @rutger-h 
A preview of the logo with 8px padding is now on https://v10.test.valtimo.nl/

Comment (rutger-h) [reply projects/valtimo/components/src/lib/components/topbar/topbar.component.scss:36]: Ok, check. Top and bottom are ok now. Left should be in line with the menu, 10px
Screenshot 2023-01-09 at 09 26 25

Comment (Klaas-Ritense) [reply projects/valtimo/components/src/lib/components/topbar/topbar.component.scss]: I've hange the padding-left to 16 px to make it inline with the menu. A preview is now deployed

**Review by rutger-h — APPROVED** [review]

**Pull request merged**